docs/changelog.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 ~~~~~~~~~
 
+0.15.0 (April 2023)
+-------------------
+
+* Darkmode added to the documentation
+* Updated pytket version requirement to 1.13.2
+* Default compilation passes update to correctly track initial and final maps during compilation
```

Comment:
*updated

Reply:
Like this? https://github.com/CQCL/pytket-quantinuum/pull/125

(Can commit directly to main without PR)